src/services/notificationService.ts

```diff
@@ -47,32 +47,32 @@ const GCMTemplate: INotificationTemplate = {
 };
 
 export default class NotificationService {
+  private notificationHubService: azure.NotificationHubService;
   constructor(
     private readonly hubName: string,
-    private readonly endpointOrConnectionString: string
-  ) {}
+    private readonly endpointOrConnectionString: string,
+    private readonly allowMultipleSessions: ALLOW_MULTIPLE_SESSIONS_OPTION
```

Comment:
```suggestion
    private readonly notificationServiceOptions: NotificationServiceOptions
```

(even if atm the type NotificationServiceOptions equals ALLOW_MULTIPLE_SESSIONS_OPTION)

Reply:
Fixed in aea043b